### PR TITLE
Added a html question on empty tag

### PR DESCRIPTION
**Pull request by Utpalkant6204**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/html-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,16 @@
 const htmlQuiz = [
+  {
+    Question:
+    "What is meant by an empty tag in HTML?",
+    Answer: "An empty tag doesn't require a closing tag",
+    Distractor1: "There is no such concept of empty tag in HTML",
+    Distractor2: "An empty tag cannot having any content within it",
+    Distractor3: "None of these",
+    Explanation:
+      "In HTML, empty tags are those tags that don’t require a closing tag for completion.",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:10]:
```suggestion
      "In HTML, empty tags are those tags that don’t require a closing tag.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by Utpalkant6204** [comment]
Please merge if it don't need any change.

**Pull request merged**